zanata-war/src/main/webapp/WEB-INF/urlrewrite.xml

```diff
@@ -293,6 +293,15 @@
     <to last="true">/webtrans/Application.seam$1</to>
   </rule>
 
+  <!-- zanata-assets resource -->
+  <!-- JSF serves zanata-assets with suffix of .seam,
+      this is to make sure any reference to zanata-assets without .seam can access to the resource.
+      e.g. /assets/style.css are redirect to /assets/style.css.seam -->
+  <rule>
+    <from casesensitive="true">^/javax.faces.resource/assets/(.*)(?&lt;!.seam)$</from>
+    <to last="true">/javax.faces.resource/assets/$1.seam</to>
```

Comment:
> @seanf our zanata-assets.jar is not generated from webjar lib. It using pom.xml I created in zanata-assets repo.
Should we even have the webjar reference in our url and package structure?

Well, perhaps it is a bad idea to use `/webjar` since the zanata-assets artifact isn't currently in the maven group `org.webjar`.  I was hoping we could make it transparent, in case we switch to bower packaging and let webjars.org package zanata-assets for us.  But perhaps that isn't a good enough reason to borrow the webjar path ahead of time.

Okay then, don't use `/webjar` for it, but I do think we need something generic so that we don't need multiple urlrewrite rules when we package other things this way.

> And also with the /{version} as part of the structure, I think it will be harder to be referenced from html page given that we need to pass the version number around.

We could use some code to help us build the correct URLs.  If we don't put the version into the URL, it may be difficult to get the caching strategy right.

But we can tackle the versioning problem another time, so feel free to leave it out for now.